docs/ops/quantization/FakeQuantize_1.md

```diff
@@ -29,37 +29,41 @@ else:
   * **Description**: *levels* is the number of quantization levels (e.g. 2 is for binarization, 255/256 is for int8 quantization)
   * **Range of values**: an integer greater than or equal to 2
   * **Type**: `int`
-  * **Default value**: None
   * **Required**: *yes*
 
 * *auto_broadcast*
 
   * **Description**: specifies rules used for auto-broadcasting of input tensors.
   * **Range of values**:
     * *none* - no auto-broadcasting is allowed, all input shapes should match
-    * *numpy* - numpy broadcasting rules, aligned with ONNX Broadcasting. Description is available in <a href="https://github.com/onnx/onnx/blob/master/docs/Broadcasting.md">ONNX docs</a>
-    * *pdpd* - PaddlePaddle-style implicit broadcasting.
+    * *numpy* - numpy broadcasting rules, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md),
+    * *pdpd* - PaddlePaddle-style implicit broadcasting, description is available in [Broadcast Rules For Elementwise Operations](../broadcast_rules.md).
   * **Type**: string
   * **Default value**: "numpy"
   * **Required**: *no*
 
 **Inputs**:
 
-*   **1**: `X` - multidimensional input tensor of floating type to be quantized. Required.
+*   **1**: `X` - tensor of type *T_F* and arbitrary shape. **Required.**
 
-*   **2**: `input_low` - minimum limit for input value. The shape must be broadcastable to the shape of *X*. Required.
+*   **2**: `input_low` - tensor of type *T_F* with minimum limit for input value. The shape must be broadcastable to the shape of *X*. **Required.**
 
-*   **3**: `input_high` - maximum limit for input value. Can be the same as `input_low` for binarization. The shape must be broadcastable to the shape of *X*. Required.
+*   **3**: `input_high` - tensor of type *T_F* with maximum limit for input value. Can be the same as `input_low` for binarization. The shape must be broadcastable to the shape of *X*. **Required.**
```

Comment:
What will be if `input_high < input_low`? And the same question is for `output_high` and `output_low`.

Reply:
Also, it seems that the formula above does not work for a case `input_high = input_low`. Please clarify these cases

Reply:
From my perspective this is UB.

Reply:
`input_high = input_low` is acceptable from the specification but formula does not work.

Reply:
My bad. it must work because `x` is clipped to `input_high = input_low` at the first step

Reply:
I would say that the first two cases in the formula really handle `input_high == input_low` and `input_high < input_low`.

- For `input_high == input_low` it would be like an edge, `output_low` for equal or lower, `output_high` for higher as `min` and `max` will return the same number.

- For `input_high < input_low` it takes `min` as low and `max` as high anyway.
(maybe we could add `abs` (input_high - input_low) in the last `else` to be consistent)

- For `output_high < output_low` there is no `min` `max` but this is how the current implementation works.



```
if x <= min(input_low, input_high):
    output = output_low
elif x > max(input_low, input_high):
    output = output_high
else:
    # input_low < x <= input_high
    output = round((x - input_low) / (input_high - input_low) * (levels-1)) / (levels-1) * (output_high - output_low) + output_low
```

Reply:
`abs` is not needed because the current version of formula gives to negates in nominator and denominator that reduce each other